### DISCUSSION TITLE
webdav文件列表正常、传输文件出错

### DISCUSSION THREAD

**Discussion by wxcatv** [AlistGo/alist#302] [Q&A] [answered]
用winscp或者Cyberduck登录webdav，文件列表正常，但传输文件时出现Could not parse redirect destination URL

**Comment by xhofe** [comment, marked as answer]
文件下载没有从服务器中转，而是302到真实链接，所以请使用能够解析redirect url的WebDAV软件，我并不知道winscp所需要的链接格式。

**Reply by wxcatv** [reply]
kodi能够解析redirect url，可以解决这个问题，谢谢！
